jeevesdb/JeevesModel.py

```diff
@@ -353,12 +350,12 @@ def __setattr__(self, name, value):
                         if hasattr(self, '_meta') else []
         if name in field_names and \
             name not in ('jeeves_vars', 'jeeves_id', 'id'):
-            old_val = getattr(self, name) if hasattr(self, name) else \
```

Comment:
This line caused an infinite loop for me; I'm not sure why updating Django would cause that, though, so it's possible that this was due to another issue in my setup and this fix isn't necessary.